100-sqlvm-setupkeyvault/azuredeploy.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "virtualMachineName": {
```

Comment:
Rename this to existingVirtualMachineName to set the right expectation i.e. this is a VM that needs to already be created when executing this template and will not be created by this template